cli/command/image/build.go

```diff
@@ -160,32 +141,13 @@ func NewBuildCommand(dockerCli command.Cli) *cobra.Command {
 
 	command.AddTrustVerificationFlags(flags, &options.untrusted, dockerCli.ContentTrustEnabled())
 
-	flags.StringVar(&options.platform, "platform", os.Getenv("DOCKER_DEFAULT_PLATFORM"), "Set platform if server is multi-platform capable")
```

Comment:
This is still necessary for the classic builder, right? (Wouldn't it even still be usable with `buildx`?)

Reply:
Good point I overlooked the [annotation](https://github.com/docker/cli/blob/6e2838e18645e06f3e4b6c5143898ccc44063e3b/cli/command/image/build.go#L164) that specifies compatibility with legacy builder.